app/views/onestring.php

```diff
@@ -23,13 +23,17 @@
     <tr>
         <th>Locale</th>
         <th>Translation</th>
+        <th>&nbsp;</th>
     </tr>
     <?php
+    $reference_locale = Project::getReferenceLocale($repo);
```

Comment:
It's used inside a foreach, which means we call the method once for every translation.